Quaver.Shared/Screens/ConfirmDialog.cs

```diff
@@ -0,0 +1,240 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * Copyright (c) Swan & The Quaver Team <support@quavergame.com>.
+*/
+
+using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Input;
+using Quaver.Shared.Assets;
+using Quaver.Shared.Graphics;
+using Wobble.Graphics;
+using Wobble.Graphics.Animations;
+using Wobble.Graphics.Primitives;
+using Wobble.Graphics.Sprites;
+using Wobble.Graphics.UI.Buttons;
+using Wobble.Graphics.UI.Dialogs;
+using Wobble.Input;
+using Wobble.Logging;
+using Wobble.Window;
+
+namespace Quaver.Shared.Screens
+{
+    public class ConfirmDialog : DialogScreen
```

Comment:
This class isn't necessary. We've already got [ConfirmCancelDialog](https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Graphics/Dialogs/ConfirmCancelDialog.cs)